### PR TITLE
Mykola/fourth block

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: dist/js/q7.js [hunk 1]
```diff
@@ -29,6 +61,44 @@ const hideActiveItem = () => {
...
+    const combined = [].concat(
+        Array.prototype.slice.call(smallCircles),
+        Array.prototype.slice.call(bigCircles)
+    )
+    makeVisible(combined)
+}
+
+// Header
+// Wrapper
+const header_slider = document.querySelector('.slider')
```
Comment (StadniukYevhen) [inline dist/js/q7.js]: rename to camelCase

File: dist/js/q7.js [hunk 2]
```diff
@@ -29,6 +61,44 @@ const hideActiveItem = () => {
...
+
+// Controls
+const controls_collection = document.querySelectorAll('.control')
```
Comment (StadniukYevhen) [inline dist/js/q7.js]: rename to camelCase

File: dist/js/templates/classHandlers.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+// Animation for svg circles
+const getCollectionElement = (element, index) => element[index]
+
+const toggleClass = (previousElement, currentElement) => {
+    const clickedSvg = document.querySelectorAll('.clicked')
+
+    previousElement.classList.contains('current') && previousElement.classList.remove('current')
+    currentElement.classList.add('current')
+    clickedSvg && clickedSvg.forEach(element => element.classList.remove('clicked'))
```
Comment (StadniukYevhen) [inline dist/js/templates/classHandlers.js]: move function to the next line

File: dist/js/q7.js (hunk 1 above)
Comment (nmelnychenko) [inline dist/js/q7.js]: done

File: dist/js/q7.js (hunk 2 above)
Comment (nmelnychenko) [inline dist/js/q7.js]: done

File: dist/js/templates/classHandlers.js (hunk 3 above)
Comment (nmelnychenko) [inline dist/js/templates/classHandlers.js]: done

**Pull request merged**